server/config/toml.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/spf13/viper"
+	cmn "github.com/tendermint/tendermint/libs/common"
+)
+
+var configTemplate *template.Template
+
+func init() {
+	var err error
+	if configTemplate, err = template.New("cosmosConfigFileTemplate").Parse(defaultConfigTemplate); err != nil {
```

Comment:
Can we create `template.New("cosmosConfigFileTemplate")` into its own variable `tmpl` just to make things easier to read?

Reply:
Can't quite get it. You want to store `"cosmosConfigFileTemplate"` into a variable?

Reply:
```go
var err error
tmpl := template.New("cosmosConfigFileTemplate")

configTemplate, err = tmpl.Parse(defaultConfigTemplate)
if err != nil {
    // ...
}
```

Reply:
ACKd